backend/kale/common/podutils.py

```diff
@@ -300,3 +306,55 @@ def compute_component_id(pod):
     component_id = component_name + "@sha256=" + component_spec_digest
     log.info("Computed component ID: %s", component_id)
     return component_id
+
+
+def get_snapshotclasses(label_selector=""):
+    """List snapshotclasses."""
+    co_client = k8sutils.get_co_client()
+
+    snapshotclasses = co_client.list_cluster_custom_object(
+        group="snapshot.storage.k8s.io",
+        version="v1beta1",
+        plural="volumesnapshotclasses",
+        label_selector=label_selector)
+    return snapshotclasses
```

Comment:
Maybe return `snapshotclasses.get("items", [])` here? So you don't have to index the result in `list_snapshotclass_storage_provisioners`